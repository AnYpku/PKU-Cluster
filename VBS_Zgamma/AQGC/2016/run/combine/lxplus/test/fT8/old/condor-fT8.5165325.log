000 (5165325.000.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.001.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.002.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.003.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.004.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.005.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.006.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.007.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.008.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.009.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.010.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.011.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
000 (5165325.012.000) 10/06 20:19:39 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_1819_42dc_3>
...
040 (5165325.000.000) 10/06 21:15:51 Started transferring input files
	Transferring to host: <188.184.212.154:9618?addrs=188.184.212.154-9618+[2001-1458-301-10c--100-85]-9618&noUDP&sock=10509_4a84_58903>
...
040 (5165325.000.000) 10/06 21:15:51 Finished transferring input files
...
001 (5165325.000.000) 10/06 21:15:52 Job executing on host: <188.184.212.154:9618?addrs=188.184.212.154-9618+[2001-1458-301-10c--100-85]-9618&noUDP&sock=20662_fdec_3>
...
006 (5165325.000.000) 10/06 21:16:00 Image size of job updated: 139472
	82  -  MemoryUsage of job (MB)
	83268  -  ResidentSetSize of job (KB)
...
040 (5165325.001.000) 10/06 21:16:29 Started transferring input files
	Transferring to host: <188.184.138.212:9618?addrs=188.184.138.212-9618&noUDP&sock=2903_2d0f_28403>
...
040 (5165325.003.000) 10/06 21:16:29 Started transferring input files
	Transferring to host: <188.184.138.212:9618?addrs=188.184.138.212-9618&noUDP&sock=2903_2d0f_28404>
...
040 (5165325.002.000) 10/06 21:16:29 Started transferring input files
	Transferring to host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=18465_0348_73070>
...
040 (5165325.001.000) 10/06 21:16:29 Finished transferring input files
...
040 (5165325.003.000) 10/06 21:16:29 Finished transferring input files
...
040 (5165325.002.000) 10/06 21:16:29 Finished transferring input files
...
040 (5165325.004.000) 10/06 21:16:30 Started transferring input files
	Transferring to host: <128.142.168.150:9618?addrs=128.142.168.150-9618+[2001-1458-301-be--100-f]-9618&noUDP&sock=4346_e66a_1699>
...
040 (5165325.004.000) 10/06 21:16:30 Finished transferring input files
...
040 (5165325.005.000) 10/06 21:16:30 Started transferring input files
	Transferring to host: <128.142.168.150:9618?addrs=128.142.168.150-9618+[2001-1458-301-be--100-f]-9618&noUDP&sock=4346_e66a_1700>
...
040 (5165325.005.000) 10/06 21:16:30 Finished transferring input files
...
040 (5165325.006.000) 10/06 21:16:30 Started transferring input files
	Transferring to host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=18465_0348_73071>
...
040 (5165325.006.000) 10/06 21:16:30 Finished transferring input files
...
001 (5165325.003.000) 10/06 21:16:30 Job executing on host: <188.184.138.212:9618?addrs=188.184.138.212-9618&noUDP&sock=1930_65df_3>
...
001 (5165325.001.000) 10/06 21:16:30 Job executing on host: <188.184.138.212:9618?addrs=188.184.138.212-9618&noUDP&sock=1930_65df_3>
...
001 (5165325.002.000) 10/06 21:16:30 Job executing on host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=2330_c3c1_3>
...
040 (5165325.008.000) 10/06 21:16:31 Started transferring input files
	Transferring to host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=18465_0348_73072>
...
040 (5165325.007.000) 10/06 21:16:31 Started transferring input files
	Transferring to host: <188.184.212.154:9618?addrs=188.184.212.154-9618+[2001-1458-301-10c--100-85]-9618&noUDP&sock=10509_4a84_58904>
...
040 (5165325.009.000) 10/06 21:16:31 Started transferring input files
	Transferring to host: <188.184.212.154:9618?addrs=188.184.212.154-9618+[2001-1458-301-10c--100-85]-9618&noUDP&sock=10509_4a84_58905>
...
040 (5165325.008.000) 10/06 21:16:31 Finished transferring input files
...
040 (5165325.007.000) 10/06 21:16:31 Finished transferring input files
...
040 (5165325.009.000) 10/06 21:16:31 Finished transferring input files
...
001 (5165325.004.000) 10/06 21:16:31 Job executing on host: <128.142.168.150:9618?addrs=128.142.168.150-9618+[2001-1458-301-be--100-f]-9618&noUDP&sock=2356_9938_23>
...
001 (5165325.005.000) 10/06 21:16:31 Job executing on host: <128.142.168.150:9618?addrs=128.142.168.150-9618+[2001-1458-301-be--100-f]-9618&noUDP&sock=2356_9938_23>
...
001 (5165325.006.000) 10/06 21:16:31 Job executing on host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=2330_c3c1_3>
...
040 (5165325.010.000) 10/06 21:16:31 Started transferring input files
	Transferring to host: <188.184.138.212:9618?addrs=188.184.138.212-9618&noUDP&sock=2903_2d0f_28405>
...
040 (5165325.010.000) 10/06 21:16:31 Finished transferring input files
...
001 (5165325.009.000) 10/06 21:16:32 Job executing on host: <188.184.212.154:9618?addrs=188.184.212.154-9618+[2001-1458-301-10c--100-85]-9618&noUDP&sock=20662_fdec_3>
...
001 (5165325.007.000) 10/06 21:16:32 Job executing on host: <188.184.212.154:9618?addrs=188.184.212.154-9618+[2001-1458-301-10c--100-85]-9618&noUDP&sock=20662_fdec_3>
...
001 (5165325.008.000) 10/06 21:16:32 Job executing on host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=2330_c3c1_3>
...
001 (5165325.010.000) 10/06 21:16:32 Job executing on host: <188.184.138.212:9618?addrs=188.184.138.212-9618&noUDP&sock=1930_65df_3>
...
040 (5165325.011.000) 10/06 21:16:36 Started transferring input files
	Transferring to host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=18465_0348_73073>
...
040 (5165325.011.000) 10/06 21:16:36 Finished transferring input files
...
001 (5165325.011.000) 10/06 21:16:37 Job executing on host: <188.184.186.64:9618?addrs=188.184.186.64-9618+[2001-1458-301-f8--100-28]-9618&noUDP&sock=2330_c3c1_3>
...
040 (5165325.012.000) 10/06 21:16:37 Started transferring input files
	Transferring to host: <128.142.117.76:9618?addrs=128.142.117.76-9618+[2001-1458-301-ae--100-518]-9618&noUDP&sock=18586_44ad_11185>
...
040 (5165325.012.000) 10/06 21:16:37 Finished transferring input files
...
001 (5165325.012.000) 10/06 21:16:38 Job executing on host: <128.142.117.76:9618?addrs=128.142.117.76-9618+[2001-1458-301-ae--100-518]-9618&noUDP&sock=18451_6e06_3>
...
006 (5165325.004.000) 10/06 21:16:39 Image size of job updated: 112732
	83  -  MemoryUsage of job (MB)
	84324  -  ResidentSetSize of job (KB)
...
006 (5165325.002.000) 10/06 21:16:39 Image size of job updated: 84068
	81  -  MemoryUsage of job (MB)
	82440  -  ResidentSetSize of job (KB)
...
006 (5165325.003.000) 10/06 21:16:39 Image size of job updated: 92104
	88  -  MemoryUsage of job (MB)
	89172  -  ResidentSetSize of job (KB)
...
006 (5165325.001.000) 10/06 21:16:39 Image size of job updated: 92584
	89  -  MemoryUsage of job (MB)
	90524  -  ResidentSetSize of job (KB)
...
006 (5165325.008.000) 10/06 21:16:40 Image size of job updated: 83216
	81  -  MemoryUsage of job (MB)
	82040  -  ResidentSetSize of job (KB)
...
006 (5165325.009.000) 10/06 21:16:40 Image size of job updated: 113344
	81  -  MemoryUsage of job (MB)
	82444  -  ResidentSetSize of job (KB)
...
006 (5165325.007.000) 10/06 21:16:40 Image size of job updated: 109968
	82  -  MemoryUsage of job (MB)
	83048  -  ResidentSetSize of job (KB)
...
006 (5165325.005.000) 10/06 21:16:40 Image size of job updated: 115188
	85  -  MemoryUsage of job (MB)
	86808  -  ResidentSetSize of job (KB)
...
006 (5165325.006.000) 10/06 21:16:40 Image size of job updated: 87548
	84  -  MemoryUsage of job (MB)
	85568  -  ResidentSetSize of job (KB)
...
006 (5165325.010.000) 10/06 21:16:41 Image size of job updated: 146284
	89  -  MemoryUsage of job (MB)
	90848  -  ResidentSetSize of job (KB)
...
006 (5165325.011.000) 10/06 21:16:46 Image size of job updated: 138464
	82  -  MemoryUsage of job (MB)
	83328  -  ResidentSetSize of job (KB)
...
006 (5165325.012.000) 10/06 21:16:47 Image size of job updated: 104284
	85  -  MemoryUsage of job (MB)
	86088  -  ResidentSetSize of job (KB)
...
006 (5165325.000.000) 10/06 21:21:00 Image size of job updated: 201656
	196  -  MemoryUsage of job (MB)
	199780  -  ResidentSetSize of job (KB)
...
006 (5165325.004.000) 10/06 21:21:39 Image size of job updated: 251936
	219  -  MemoryUsage of job (MB)
	223524  -  ResidentSetSize of job (KB)
...
006 (5165325.002.000) 10/06 21:21:40 Image size of job updated: 193156
	188  -  MemoryUsage of job (MB)
	191608  -  ResidentSetSize of job (KB)
...
006 (5165325.003.000) 10/06 21:21:40 Image size of job updated: 228484
	221  -  MemoryUsage of job (MB)
	225540  -  ResidentSetSize of job (KB)
...
006 (5165325.001.000) 10/06 21:21:40 Image size of job updated: 227628
	221  -  MemoryUsage of job (MB)
	225560  -  ResidentSetSize of job (KB)
...
006 (5165325.008.000) 10/06 21:21:40 Image size of job updated: 192820
	188  -  MemoryUsage of job (MB)
	191668  -  ResidentSetSize of job (KB)
...
006 (5165325.009.000) 10/06 21:21:40 Image size of job updated: 231892
	197  -  MemoryUsage of job (MB)
	200984  -  ResidentSetSize of job (KB)
...
006 (5165325.007.000) 10/06 21:21:40 Image size of job updated: 227900
	197  -  MemoryUsage of job (MB)
	200968  -  ResidentSetSize of job (KB)
...
006 (5165325.006.000) 10/06 21:21:41 Image size of job updated: 194232
	188  -  MemoryUsage of job (MB)
	192212  -  ResidentSetSize of job (KB)
...
006 (5165325.005.000) 10/06 21:21:41 Image size of job updated: 251656
	219  -  MemoryUsage of job (MB)
	223268  -  ResidentSetSize of job (KB)
...
006 (5165325.010.000) 10/06 21:21:42 Image size of job updated: 281000
	221  -  MemoryUsage of job (MB)
	225556  -  ResidentSetSize of job (KB)
...
006 (5165325.011.000) 10/06 21:21:47 Image size of job updated: 246504
	187  -  MemoryUsage of job (MB)
	191356  -  ResidentSetSize of job (KB)
...
006 (5165325.012.000) 10/06 21:21:48 Image size of job updated: 226344
	204  -  MemoryUsage of job (MB)
	208148  -  ResidentSetSize of job (KB)
...
040 (5165325.004.000) 10/06 21:24:22 Started transferring output files
...
040 (5165325.004.000) 10/06 21:24:22 Finished transferring output files
...
005 (5165325.004.000) 10/06 21:24:22 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:07:47, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:07:47, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	389  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	389  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    27           3   3321633 
	   Memory (MB)          :   219        2000      2000 
...
006 (5165325.005.000) 10/06 21:24:23 Image size of job updated: 286580
	219  -  MemoryUsage of job (MB)
	223268  -  ResidentSetSize of job (KB)
...
040 (5165325.005.000) 10/06 21:24:23 Started transferring output files
...
040 (5165325.005.000) 10/06 21:24:23 Finished transferring output files
...
005 (5165325.005.000) 10/06 21:24:23 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:07:48, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:07:48, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	390  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	390  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    27           3   3321632 
	   Memory (MB)          :   219        2000      2000 
...
040 (5165325.003.000) 10/06 21:24:29 Started transferring output files
...
040 (5165325.003.000) 10/06 21:24:29 Finished transferring output files
...
005 (5165325.003.000) 10/06 21:24:30 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:07:54, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:07:54, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	390  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	390  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    27           3    332184 
	   Memory (MB)          :   221        2000      2000 
...
040 (5165325.001.000) 10/06 21:24:35 Started transferring output files
...
040 (5165325.001.000) 10/06 21:24:35 Finished transferring output files
...
005 (5165325.001.000) 10/06 21:24:35 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:08:00, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:08:00, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	558273  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	558273  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :   572           3    332184 
	   Memory (MB)          :   221        2000      2000 
...
006 (5165325.010.000) 10/06 21:24:41 Image size of job updated: 314228
	221  -  MemoryUsage of job (MB)
	225556  -  ResidentSetSize of job (KB)
...
040 (5165325.010.000) 10/06 21:24:41 Started transferring output files
...
040 (5165325.010.000) 10/06 21:24:41 Finished transferring output files
...
005 (5165325.010.000) 10/06 21:24:41 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:08:04, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:08:04, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	394  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	394  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    27           3    332183 
	   Memory (MB)          :   221        2000      2000 
...
006 (5165325.000.000) 10/06 21:26:01 Image size of job updated: 241700
	235  -  MemoryUsage of job (MB)
	239824  -  ResidentSetSize of job (KB)
...
006 (5165325.002.000) 10/06 21:26:40 Image size of job updated: 231140
	225  -  MemoryUsage of job (MB)
	229592  -  ResidentSetSize of job (KB)
...
006 (5165325.008.000) 10/06 21:26:41 Image size of job updated: 230768
	225  -  MemoryUsage of job (MB)
	229616  -  ResidentSetSize of job (KB)
...
006 (5165325.009.000) 10/06 21:26:41 Image size of job updated: 271300
	235  -  MemoryUsage of job (MB)
	240392  -  ResidentSetSize of job (KB)
...
006 (5165325.007.000) 10/06 21:26:41 Image size of job updated: 267320
	235  -  MemoryUsage of job (MB)
	240388  -  ResidentSetSize of job (KB)
...
006 (5165325.006.000) 10/06 21:26:42 Image size of job updated: 231940
	225  -  MemoryUsage of job (MB)
	229920  -  ResidentSetSize of job (KB)
...
006 (5165325.011.000) 10/06 21:26:48 Image size of job updated: 284496
	224  -  MemoryUsage of job (MB)
	229348  -  ResidentSetSize of job (KB)
...
006 (5165325.012.000) 10/06 21:26:49 Image size of job updated: 265200
	242  -  MemoryUsage of job (MB)
	247000  -  ResidentSetSize of job (KB)
...
006 (5165325.012.000) 10/06 21:28:25 Image size of job updated: 666560
	242  -  MemoryUsage of job (MB)
	247000  -  ResidentSetSize of job (KB)
...
040 (5165325.012.000) 10/06 21:28:26 Started transferring output files
...
040 (5165325.012.000) 10/06 21:28:26 Finished transferring output files
...
005 (5165325.012.000) 10/06 21:28:26 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:11:41, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:11:41, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	378  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	378  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        1         1 
	   Disk (KB)            :       27        3    217858 
	   Memory (MB)          :      242     2000      2000 
...
040 (5165325.000.000) 10/06 21:28:59 Started transferring output files
...
040 (5165325.000.000) 10/06 21:28:59 Finished transferring output files
...
005 (5165325.000.000) 10/06 21:29:00 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:12:58, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:12:58, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	32749  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	32749  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    59           3    292807 
	   Memory (MB)          :   235        2000      2000 
...
040 (5165325.007.000) 10/06 21:29:24 Started transferring output files
...
040 (5165325.007.000) 10/06 21:29:24 Finished transferring output files
...
006 (5165325.009.000) 10/06 21:29:25 Image size of job updated: 666572
	235  -  MemoryUsage of job (MB)
	240392  -  ResidentSetSize of job (KB)
...
005 (5165325.007.000) 10/06 21:29:25 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:12:44, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:12:44, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	394  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	394  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    27           3    292219 
	   Memory (MB)          :   235        2000      2000 
...
040 (5165325.009.000) 10/06 21:29:26 Started transferring output files
...
040 (5165325.009.000) 10/06 21:29:26 Finished transferring output files
...
005 (5165325.009.000) 10/06 21:29:26 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:12:44, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:12:44, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	395  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	395  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        1         1 
	   Disk (KB)            :       28        3    292219 
	   Memory (MB)          :      235     2000      2000 
...
040 (5165325.006.000) 10/06 21:31:09 Started transferring output files
...
040 (5165325.006.000) 10/06 21:31:09 Finished transferring output files
...
005 (5165325.006.000) 10/06 21:31:09 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:14:21, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:14:21, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	391  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	391  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.99        1         1 
	   Disk (KB)            :    27           3    211517 
	   Memory (MB)          :   225        2000      2000 
...
040 (5165325.002.000) 10/06 21:31:10 Started transferring output files
...
040 (5165325.002.000) 10/06 21:31:10 Finished transferring output files
...
005 (5165325.002.000) 10/06 21:31:11 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:14:22, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:14:22, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	13906  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	13906  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    40           3    211517 
	   Memory (MB)          :   225        2000      2000 
...
040 (5165325.008.000) 10/06 21:31:11 Started transferring output files
...
040 (5165325.008.000) 10/06 21:31:11 Finished transferring output files
...
005 (5165325.008.000) 10/06 21:31:12 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:14:22, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:14:22, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	395  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	395  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    27           3    211517 
	   Memory (MB)          :   225        2000      2000 
...
006 (5165325.011.000) 10/06 21:31:19 Image size of job updated: 313944
	224  -  MemoryUsage of job (MB)
	229348  -  ResidentSetSize of job (KB)
...
040 (5165325.011.000) 10/06 21:31:19 Started transferring output files
...
040 (5165325.011.000) 10/06 21:31:19 Finished transferring output files
...
005 (5165325.011.000) 10/06 21:31:19 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:14:26, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:14:26, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1050  -  Run Bytes Sent By Job
	2590  -  Run Bytes Received By Job
	1050  -  Total Bytes Sent By Job
	2590  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    28           3    211516 
	   Memory (MB)          :   224        2000      2000 
...
