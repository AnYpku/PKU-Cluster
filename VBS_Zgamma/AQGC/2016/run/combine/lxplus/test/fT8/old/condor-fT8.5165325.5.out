 <<< Combine >>> 
>>> including systematics
>>> using the Profile Likelihood test statistics modified for upper limits (Q_LHC)
>>> method used is HybridNew
>>> Used OpenSSL to get a really random seed -97421720
Computing results starting from observation (a-posteriori)

 -- Hybrid New -- 
CLs = 0.176471 +/- 0.0206499
Done in 7.75 min (cpu), 7.76 min (real)
